packages/solid/src/server/rendering.ts

```diff
@@ -569,8 +568,8 @@ export function Suspense(props: { fallback?: string; children: string }) {
     });
   function runSuspense() {
     setHydrateContext({ ...ctx, count: 0 });
-    o && cleanNode(o);
```

Comment:
`o` is always present. before 1.6.13 `clean` was checked here

Reply:
Are we sure we can just remove it? I hoisted it up for sure and that caused the issue, but it was always there. I guess I can retest the original React Query scenario.

Reply:
pre `1.6.13` the current `Owner` would get assigned as `o` so it could be `null`, now we are always a new owner so the condition is pointless.

Reply:
Also, github is showing the preview a bit weirdly. I didn't remove `cleanNode(o)`, just `o &&`

Reply:
Oh I get it. Yeah I was looking at the wrong part. I guess the trickiest part is I change this mechanism completely in 1.7. It's going to be a bit interesting to figure out how to merge this back into next. It looks like I need to do another 1.6 release given the stuff I've broken though so I will go with this for now.